### PR TITLE
Add cloud Annotation Support for .NET MAUI PdfViewer

### PR DESCRIPTION
**Pull request by manikandan18102001**
Title : UG content preparation for Cloud annotaion in .NETMAUI SfPdfViewer
Task Link : https://dev.azure.com/EssentialStudio/Mobile%20and%20Desktop/_workitems/edit/890613/

### CODE REVIEW

**Review by Deepak1799 — CHANGES_REQUESTED** [review]
Requested changes

File: MAUI/PDF-Viewer/Shapes.md [hunk 1]
```diff
@@ -157,4 +158,106 @@ void EditSelectedCicleAnnotation(Annotation selectedAnnotation)
...
+
+## Border styles for shape annotation
+
+[SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) allows you to change the shape annotations' border style. Currently, it provides support for the following border styles.
+1.	Cloudy
+2.	Solid
+
+N> By default, border style of the shape annotations is solid. The value of `BorderStyle` property is applicable to square and polygon annotations only and it does not affect other shape annotations such as circle, line, arrow, and polyline annotations.
+
+### Draw cloud shape annotation
```
[the remark this answers is not in the document]
Comment (manikandan18102001) [reply MAUI/PDF-Viewer/Shapes.md]: Addressed

File: MAUI/PDF-Viewer/Shapes.md [hunk 2]
```diff
@@ -157,4 +158,106 @@ void EditSelectedCicleAnnotation(Annotation selectedAnnotation)
...
+	// Draw polygon cloud annotation
+	pdfViewer.AnnotationMode = AnnotationMode.Polygon;
+	pdfViewer.AnnotationSettings.Rectangle.BorederStyle = BorderStyle.Cloudy;
+
+{% endhighlight %}
+{% endtabs %}
+
+### Change the selected shape annotation border style
+
+You can change the border style of the selected shape annotation programmatically by accessing the selected annotation instance. The selected annotation instance may be obtained from the [AnnotationSelected](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html#Syncfusion_Maui_PdfViewer_SfPdfViewer_AnnotationSelected) event. The following example shows how to change the border style of selected polygon annotation. Similarly, you can modify the square annotations border style.
```
[the remark this answers is not in the document]
Comment (manikandan18102001) [reply MAUI/PDF-Viewer/Shapes.md]: Changed

File: MAUI/PDF-Viewer/Shapes.md [hunk 3]
```diff
@@ -157,4 +158,106 @@ void EditSelectedCicleAnnotation(Annotation selectedAnnotation)
...
+{% highlight C# %}
+
+	// Draw polygon cloud annotation
+	pdfViewer.AnnotationMode = AnnotationMode.Polygon;
+	pdfViewer.AnnotationSettings.Rectangle.BorederStyle = BorderStyle.Cloudy;
+
+{% endhighlight %}
+{% endtabs %}
+
+### Change the selected shape annotation border style
```
[the remark this answers is not in the document]
Comment (manikandan18102001) [reply MAUI/PDF-Viewer/Shapes.md]: changed into =>"Change the border style for selected polygon annotation"

File: MAUI/PDF-Viewer/Shapes.md [hunk 4]
```diff
@@ -157,4 +158,106 @@ void EditSelectedCicleAnnotation(Annotation selectedAnnotation)
...
\ No newline at end of file
+{% endtabs %}
+
+## Border styles for shape annotation
+
+[SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) allows you to change the shape annotations' border style. Currently, it provides support for the following border styles.
+1.	Cloudy
+2.	Solid
+
+N> By default, border style of the shape annotations is solid. The value of `BorderStyle` property is applicable to square and polygon annotations only and it does not affect other shape annotations such as circle, line, arrow, and polyline annotations.
```
[the remark this answers is not in the document]
Comment (manikandan18102001) [reply MAUI/PDF-Viewer/Shapes.md]: Changed

File: MAUI/PDF-Viewer/Shapes.md [hunk 5]
```diff
@@ -5,6 +5,7 @@ description: Learn here all about Shape Annotations in Syncfusion .NET MAUI PDF
 platform: MAUI
```
[the remark this answers is not in the document]
Comment (manikandan18102001) [reply MAUI/PDF-Viewer/Shapes.md:5]: Changed into white backgroud

**Review by NavaneethS — APPROVED** [review]

**Pull request merged**